frmts/pdf/pdfio.h

```diff
@@ -54,8 +54,18 @@ class VSIPDFFileStream final : public BaseStream
     virtual void reset() override;
 #endif
 
+    static void resetNoCheckReturnValue(Stream *str)
+    {
+#if POPPLER_MAJOR_VERSION > 25 ||                                              \
+    (POPPLER_MAJOR_VERSION == 25 && POPPLER_MINOR_VERSION >= 2)
+        CPL_IGNORE_RET_VAL(str->reset());
```

Comment:
I was wondering if we should consider a macro definition for `CPL_IGNORE_RET_VAL` such that it can tolerate a `void` return....

```
#define CPL_IGNORE_RET_VAL(x) static_cast<void>(x)
```

But I'm guessing this situation is unusual enough that it's not worth it to change anything.

Reply:
I have some vague recollection that one of the analyzers didn't like a cast to void (but I may be wrong)